lib/core/tree.js

```diff
@@ -149,8 +149,38 @@ class TernarySearchTree {
 
 const tree = new TernarySearchTree()
 
-for (let i = 0; i < wellknownHeaderNames.length; ++i) {
-  const key = headerNameLowerCasedRecord[wellknownHeaderNames[i]]
+// function shuffleArray (array) {
+//   for (let i = array.length - 1; i > 0; i--) {
+//     const j = Math.floor(Math.random() * (i + 1));
+//     [array[i], array[j]] = [array[j], array[i]]
+//   }
+//   return array
+// }
+// const indexes = []
+// wellknownHeaderNames.forEach((key, idx) => {
+//   indexes.push(idx)
+// })
+// randomIndexOfWellknownHeaderNames = shuffleArray(indexes)
```

Comment:
why is this commented out? Can you nmove this to a script or another js file instead?

Reply:
I think we do not want any potential instable performance, right? (even the chance is low :) ) so I take a snapshot of one randomized insertion order that works well

Reply:
Maybe I just remove them from codebase outright and put it the PR